task/deprecated-image-check/0.1/deprecated-image-check.yaml

```diff
@@ -36,6 +36,9 @@ spec:
           value: $(params.BASE_IMAGES_DIGESTS)
       script: |
         #!/usr/bin/env bash
+        set -euo pipefail
+        trap 'echo "Script errored at line ${LINENO}: ${BASH_COMMAND}"' ERR
```

Comment:
Should be trap handle_error here?

Reply:
This task references registry.access.redhat.com/ubi8/ubi-minimal, not our hacbs-test image.

Reply:
Actually we don't need it because we should not fail here

Reply:
Then we probably should use hacbs-test image.

> Actually we don't need it because we should not fail here

Why? there is many codelines that could fail on random unexpected error

Reply:
The image from `ubi-minimal` can be enough and the next step will generate `TEST_OUTPUT` according to `results.PYXIS_HTTP_CODE.path` https://github.com/redhat-appstudio/build-definitions/blob/main/task/deprecated-image-check/0.1/deprecated-image-check.yaml#L68

Reply:
We are trying to handle unexpected failures and silent hiding of errors. I don't know how you can be so sure, that with any change in future this will be working smoothly forever.

Reply:
I mean `ubi-minimal` have all command used in that step, so it is enough. So let me change to use hacbs-test